README.md

```diff
@@ -10,9 +10,9 @@
 
 <!-- Remove the content in this H2 heading after completing the steps -->
 
-The landing zone module can be used to create a fully customizable VPC environment within a single region. The four following patterns are starting templates that can be used to get started quickly with Landing Zone. These patterns are located in the [patterns](/patterns/) directory.
+The landing zone module can be used to create a fully customizable VPC environment within a single region. The five following patterns are starting templates that can be used to get started quickly with Landing Zone. These patterns are located in the [patterns](/patterns/) directory.
 
-Each of these patterns creates the following infrastructure:
+Each of these patterns (except QuickStart) creates the following infrastructure:
```

Comment:
There may be other Quickstart patterns in the future, so I would rename this:
`except QuickStart` -> `except VSI QuickStart `